docs/_pages/best-practices.md

```diff
@@ -0,0 +1,1987 @@
+---
+permalink: /best-practices/
+title: 'Modeling Best Practices'
+toc_label: 'Best Practices'
+---
+
+This Documentation will go over best practices developers of Simulato have found during their use of the tool.  Everything presented here has been found to make the modeling practice easier, as well making the models clear, concise and reusable.
+
+## File Structure
+
+When creating components, matching the system under test's file structure allows for ease of development.  It will allow anyone working with the HTML files to know which components represent that view so the components can be updated side by side with any changes to the view.  Constraining your component to only model what that is included with any given HTML will help keep models smaller, consistent with the system views, and help guide you create the component.
```

Comment:
`what that` -> `what` ?

Reply:
`and help guide you create the components` -> something more grammatically sensible